testament/testament.nim

```diff
@@ -66,6 +69,12 @@ type
     spec: TSpec
     startTime: float
 
+proc toStr(a: TResults): string =
```

Comment:
*sigh* Sure, let's dump EVERYTHING ALWAYS because human brains have infinite capacities wading through megabytes of junk in logfiles, it's a great fun that never gets tiresome.